base/console/src/main/java/com/netscape/admin/certsrv/Console.java

```diff
@@ -1577,6 +1580,7 @@ public static void printHelp() {
         System.out.println();
         System.out.println(" -f <file>      Capture stderr and stdout to file.");
         System.out.println(" -D <options>   Debug options.");
+        System.out.println(" -d <nssdb path>   path to nssdb; if not specified, default is .dogtag-idm-console or .redhat-idm-console, depending on the platform");
```

Comment:
Could we do something like this?

    String default_nssdb_path = FilePreferenceManager.getHomePath();
    System.out.println(" -d <nssdb path>   path to nssdb (default: " + default_nssdb_path + ")");

See https://github.com/dogtagpki/idm-console-framework/blob/master/src/main/java/com/netscape/management/client/util/UtilConsoleGlobals.java#L126.